### PR TITLE
Fix a bunch of files with pylint disabled

### PR DESCRIPTION
**Pull request by willbarrett**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [X] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY

Re-enable pylint on a number of files with pylint globally disabled.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

No UI changes.

### TEST PLAN

- [ ] Run lint
- [ ] Run tests

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS
@john-bodley

### CODE REVIEW

File: superset/jinja_context.py [hunk 1]
```diff
@@ -273,5 +272,5 @@ class HiveTemplateProcessor(PrestoTemplateProcessor):
 
 
 def get_template_processor(database, table=None, query=None, **kwargs):
-    TP = template_processors.get(database.backend, BaseTemplateProcessor)
-    return TP(database=database, table=table, query=query, **kwargs)
+    tp = template_processors.get(database.backend, BaseTemplateProcessor)
```
Comment (etr2460) [inline superset/jinja_context.py]: why not name `template_processor`?

File: superset/sql_lab.py [hunk 2]
```diff
@@ -175,12 +173,11 @@ def get_sql_results(
 
 def execute_sql_statement(sql_statement, query, user_name, session, cursor):
     """Executes a single SQL statement"""
-    query_id = query.id
     database = query.database
     db_engine_spec = database.db_engine_spec
     parsed_query = ParsedQuery(sql_statement)
     sql = parsed_query.stripped()
-    SQL_MAX_ROWS = app.config["SQL_MAX_ROW"]
+    sql_max_rows = app.config["SQL_MAX_ROW"]
```
Comment (etr2460) [inline superset/sql_lab.py]: should this be pulled out of the function to the top of the file? Then we keep the same var name but don't need to refetch from the app config each time

File: superset/sql_lab.py [hunk 3]
```diff
@@ -202,15 +199,15 @@ def execute_sql_statement(sql_statement, query, user_name, session, cursor):
...
+        if sql_max_rows and (not query.limit or query.limit > sql_max_rows):
+            query.limit = sql_max_rows
         if query.limit:
             sql = database.apply_limit_to_sql(sql, query.limit)
 
     # Hook to allow environment-specific mutation (usually comments) to the SQL
-    SQL_QUERY_MUTATOR = config["SQL_QUERY_MUTATOR"]
-    if SQL_QUERY_MUTATOR:
-        sql = SQL_QUERY_MUTATOR(sql, user_name, security_manager, database)
+    sql_query_mutator = config["SQL_QUERY_MUTATOR"]
```
Comment (etr2460) [inline superset/sql_lab.py]: same comment here

**Review by etr2460 — COMMENTED** [review]
a couple suggestions

File: superset/sql_lab.py [hunk 4]
```diff
@@ -14,14 +14,16 @@
...
 import logging
 import uuid
 from contextlib import closing
 from datetime import datetime
 from sys import getsizeof
 from typing import Optional, Tuple, Union
 
+# pylint and isort disagree on the correct import order.
+# Let's have isort win.
+# pylint: disable=ungrouped-imports
```
Comment (etr2460) [inline superset/sql_lab.py]: should we add this to a base pylint rule disable? so that we don't need it on a bunch of files?

**Review by etr2460 — APPROVED** [review]
assuming pylint and tests pass, lgtm.

one comment, but not required to address in this pass

File: superset/sql_lab.py (hunk 4 above)
Comment (john-bodley) [reply superset/sql_lab.py]: I’m a little perflexed as to why `pylint` is complaining here.

Comment (willbarrett) [reply superset/sql_lab.py]: Yup. Not a bad idea. Will update here before I forget.

**Pull request merged**